Frameworks/CoreText/CTFrame.mm

```diff
@@ -109,12 +109,12 @@ void CTFrameDraw(CTFrameRef frame, CGContextRef ctx) {
         CGPoint relativePosition = frame->_lineOrigins[i];
 
         for (size_t j = 0; j < [line->_runs count]; ++j) {
-            _CTRun* curRun = [line->_runs objectAtIndex:j];
+			__CTRun* curRun = static_cast<__CTRun*>([line->_runs objectAtIndex:j]);
```

Comment:
It's already changed as part of CTLine, that will be out soon.

---
In reply to: [111453255](https://github.com/Microsoft/WinObjC/pull/2511#discussion_r111453255) [](ancestors = 111453255)